coremltools/converters/mil/frontend/torch/test/test_torch_ops.py

```diff
@@ -4590,6 +4590,30 @@ def forward(self, x):
             frontend=frontend,
         )
 
+    @pytest.mark.parametrize(
+        "compute_unit, backend, frontend",
+        itertools.product(
+            compute_units,
+            backends,
+            frontends,
+        ),
+    )
+    def test_randn(self, compute_unit, backend, frontend):
+        class TestModel(torch.nn.Module):
+            def forward(self, x):
+                noise = torch.randn(x.shape)
+                return x + noise
+
+        self.run_compare_torch(
+            (1, 3, 16, 16),
+            TestModel(),
+            compute_unit=compute_unit,
+            backend=backend,
+            frontend=frontend,
+            atol=100.0,  # Don't verify numerical results due to randomness.
```

Comment:
Would it make sense to have a new parameter, something like `skip_numerical_check` instead?

Reply:
Yeah you read my mind :)
I was debating when I used this `atol=100.0` trick. I found that this `skip_numerical_check` need to be passed several layers deep down, and considering it's only needed for rand op, I tend to keep using the existing `atol` to work it around. We can definitely re-visit it if there are more ops in the future that need to skip numerical verification.